### PR TITLE
Update actions-plan-preview image to 1.8.0

### PR DESCRIPTION
**Pull request by gkuga**
**What this PR does / why we need it**:
* Update the image version to reflect #3513 change to the image.

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: tool/actions-plan-preview/DOCKER_BUILD [hunk 1]
```diff
@@ -1,2 +1,2 @@
-version: 1.7.2
+version: 1.7.3
```
Comment (nghialv) [inline tool/actions-plan-preview/DOCKER_BUILD:1]: @gkuga Thank you. Could you update this to `1.8.0`?

Comment (gkuga) [inline tool/actions-plan-preview/DOCKER_BUILD:1]:
```suggestion
version: 1.8.0
```

Comment (gkuga) [reply tool/actions-plan-preview/DOCKER_BUILD]: I updated the version to 1.8.0 👍

**Review by nghialv — APPROVED** [review]
Thank you.

**Review by khanhtc1202 — APPROVED** [review]

**Pull request merged**